plugins/Essentials/config.yml

```diff
@@ -20,10 +20,10 @@
 ############################################################
 
 # A color code between 0-9 or a-f. Set to 'none' to disable.
-ops-name-color: '4'
+ops-name-color: '&6&o'
```

Comment:
The default config does not contain an ampersand, hinting that this config key only accepts one color code. Of course, the comment immediately above this states this as well. You will need to fix this @NexusHD